create/using-manual-approval.adoc

```diff
@@ -0,0 +1,39 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="using-manual-approval"]
+= Using manual approval in {pipelines-shortname}
+:context: using-manual-approval
+
+toc::[]
+
+You can specify a manual approval task in a pipeline. When the pipeline reaches this task, it pauses and awaits approval from one or several users. If any of the users chooses to rejects the task instead of approving it, the pipeline fails.
```

Comment:
We should mention some where that currently only users from Openshift are supported

Reply:
done